src/explore-education-statistics-admin/src/pages/release/footnotes/__tests__/ReleaseFootnotesPage.test.tsx

```diff
@@ -0,0 +1,279 @@
+import ReleaseFootnotesPage from '@admin/pages/release/footnotes/ReleaseFootnotesPage';
+import {
+  releaseFootnotesRoute,
+  ReleaseRouteParams,
+} from '@admin/routes/releaseRoutes';
+import _footnoteService, {
+  Footnote,
+  FootnoteMeta,
+} from '@admin/services/footnoteService';
+import _permissionService from '@admin/services/permissionService';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { createMemoryHistory } from 'history';
+import React from 'react';
+import { generatePath, Route, Router } from 'react-router-dom';
+
+jest.mock('@admin/services/footnoteService');
+const footnoteService = _footnoteService as jest.Mocked<
+  typeof _footnoteService
+>;
+
+jest.mock('@admin/services/permissionService');
+const permissionService = _permissionService as jest.Mocked<
+  typeof _permissionService
+>;
+
+describe('ReleaseFootnotesPage', () => {
+  const testFootnoteMeta: FootnoteMeta = {
+    subjects: {
+      'test-subject-1': {
+        subjectId: 'test-subject-1',
+        subjectName: 'Test subject 1',
+        indicators: {},
+        filters: {},
+      },
+    },
+  };
+
+  const testFootnotes: Footnote[] = [
+    { id: 'footnote-1', content: 'Footnote 1 content', subjects: {} },
+    { id: 'footnote-2', content: 'Footnote 2 content', subjects: {} },
+    { id: 'footnote-3', content: 'Footnote 3 content', subjects: {} },
+  ];
+
+  describe('with footnotes', () => {
+    beforeEach(() => {
+      permissionService.canUpdateRelease.mockResolvedValue(true);
+      footnoteService.getFootnoteMeta.mockResolvedValue(testFootnoteMeta);
+      footnoteService.getFootnotes.mockResolvedValue(testFootnotes);
+      renderPage();
+    });
+
+    test('renders the page with footnotes', async () => {
+      await waitFor(() => {
+        expect(screen.getByText('Footnotes')).toBeInTheDocument();
+      });
+
+      expect(
+        screen.getByRole('link', { name: 'Create footnote' }),
+      ).toBeInTheDocument();
+      expect(
+        screen.getByRole('button', { name: 'Reorder footnotes' }),
+      ).toBeInTheDocument();
+
+      const footNote1 = within(
+        screen.getByTestId('Footnote - Footnote 1 content'),
+      );
+      expect(footNote1.getByText('Footnote 1 content')).toBeInTheDocument();
+      expect(
+        footNote1.getByRole('link', { name: 'Edit footnote' }),
```

Comment:
Don't feel super strongly about this, but might be worth changing the 'Edit footnote' button to use link styling and the 'Delete footnote' button to use the `ButtonText` component (using a warning variant).

Currently, when there's a bunch of footnotes, the page will have lots of primary buttons which feels a bit chaotic. We usually use primary buttons to emphasise a main user path.

Although we're quite inconsistent with this, we've recently been using link styles for buttons in 'Actions' columns of tables. It might be nice to make things a bit more consistent with this.